Importer/CsvImporter.php

```diff
@@ -64,20 +64,20 @@ public function importFile($filename, $hasHeaders = true, $flush = true)
     /**
      * @param array $csv
      * @param bool  $hasHeaders
-     * @param bool  $flush
+     * @param bool  $dryRun
      *
      * @return array|null
      * @throws DatabaseException
      */
-    public function import($csv, $hasHeaders = true, $flush = true)
+    public function import($csv, $hasHeaders = true, $dryRun = true)
     {
         if (!$this->checkHeadersAreValid($csv)) {
             return [];
         }
 
         $parsedData = $this->parser->parse($csv, $hasHeaders, $this->delimiter);
 
-        return $this->importData($parsedData, $hasHeaders, $flush);
+        return $this->importData($parsedData, $hasHeaders, $dryRun);
```

Comment:
No... As i said, it makes sense. It would not make sense as a parameter for a function called `importData`.